web/src/main/java/com/synopsys/integration/alert/web/api/job/JobConfigController.java

```diff
@@ -54,9 +54,9 @@ public JobConfigController(JobConfigActions jobConfigActions) {
         this.jobConfigActions = jobConfigActions;
     }
 
-    @GetMapping("/validate")
-    public List<JobFieldStatuses> getValidationResultsForJobs() {
-        return ResponseFactory.createContentResponseFromAction(jobConfigActions.validateAllJobs());
+    @PostMapping("/validateJobsById")
```

Comment:
Are these for the next major version? Since people are using our API, won't this break things for them?

Reply:
We have swagger documentation of endpoints, but we make it explicitly clear that these endpoints are not considered stable.